lang/de.json

```diff
@@ -1,161 +1,162 @@
 {
-    "Hello": "Hallo",
-    "User Name": "Nutzername",
-    "we've": "temos",
-    "got you covered": "hast du abgedeckt",
```

Comment:
Reviewers, please ignore the extra tab being removed from all the translation JSON files. This happened automatically because they were previously not have been formatted by a flutter code formatter